OpenNept4une.sh

```diff
@@ -763,7 +800,7 @@ print_menu() {
     echo ""
     echo -e "(${R} Q ${NC}) Quit"
     echo "=========================================================="
-    echo "Select an option by entering (1-7 / q):"
+    echo "Select an option by entering (1-6 / q):"
```

Comment:
I think the main menu still has 7 options? It seems the added option for `display_firmware` is in the Advanced menu, and wouldn't require to decrease the options from the main menu. Can you double-check please?

```suggestion
    echo "Select an option by entering (1-7 / q):"
```

Reply:
You are right must've changed it when I was updating the script to the latest version of it.